src/GafferDispatch/Dispatcher.cpp

```diff
@@ -286,16 +286,9 @@ Dispatcher::PostDispatchSignal &Dispatcher::postDispatchSignal()
 
 void Dispatcher::setupPlugs( Plug *parentPlug )
 {
-	if ( const TaskNode *node = parentPlug->ancestor<const TaskNode>() )
+	if( parentPlug->ancestor<const TaskNode>() )
```

Comment:
I think this will always be `true` won't it? My assumption is that the old check was just unnecessary belt-and-braces, and we should just put the `addChild()` call in the same scope as the one for `g_immediatePlugName`.

Reply:
Addressed in 0218368